test/controllers/spectrums_controller_test.rb

```diff
@@ -243,6 +243,29 @@ class SpectrumsControllerTest < ActionController::TestCase
     assert_redirected_to spectrum_path(assigns(:spectrum))
   end
 
+  test 'should update spectrum title' do
+    session[:user_id] = User.first.id # log in
+    patch :update, params: { id: spectrums(:one).id, spectrum: { title: "Really cool spectrum" } }
```

Comment:
aha -- ok, so this actually is being looked for as simply `title` not `spectrum: { title: "X" }`:

https://github.com/waridrox/spectral-workbench/blob/a105a5c1db55c37cbd8efa094217a48f56bb91a5/app/controllers/spectrums_controller.rb#L352-L358

However even from the user interface we're sending: 

`  Parameters: {"utf8"=>"✓", "authenticity_token"=>"nQMzQNOU8GZimN+bCpA7BqdOqJe5yLFypr79cd9iUYtf7OpoJJO9bu3ncSU2MsT30XvCGjDpkNkJvlj4kkaooA==", "spectrum"=>{"title"=>"test 3", "notes"=>""}, "id"=>"1"}`

Let's update the controller.